docs/how-to/create-a-router/configure-routes.md

```diff
@@ -0,0 +1,18 @@
+# Configure routes
+
+Routes are an essential part of your Turing Router setup. Each route is defined by its ID and the endpoint. A route can be your deployed ML model, exposed via HTTP interface, or an arbitrary non-ML web-service. Each router should be configured with at least one route and each route's ID should be unique among other routes of this router.
+
+![](../../.gitbook/assets/routes_panel.png)
+
+It is also required to specify, which is default route. Each router has exactly one default route and zero or more non-default routes. Depending on the configuration of Experiment Engine and Ensembler, Turing might use a response from the default route as a final response that would be sent to the calling upstream system:
```

Comment:
which is default route -> "which route is the default route"

Reply:
ok changed